lib/mayaUsd/render/mayaToHydra/renderGlobals.cpp

```diff
@@ -920,11 +938,25 @@ MObject MtohRenderGlobals::CreateAttributes(const GlobalParams& params)
                     attr.defaultValue.UncheckedGet<GfVec4f>(),
                     userDefaults);
             } else if (attr.defaultValue.IsHolding<TfToken>()) {
-                _CreateStringAttribute(
-                    node,
-                    filter.mayaString(),
-                    attr.defaultValue.UncheckedGet<TfToken>().GetString(),
-                    userDefaults);
```

Comment:
This is where I'd put something like:
```cpp
// If this attribute type has AllowedTokens set, we treat it as an enum instead, so that only valid values are available.```